FlowCal/io.py

```diff
@@ -1160,11 +1163,88 @@ def hist_bin_edges(self, channels=None):
         # Get numerical indices of channels
         channels = self._name_to_index(channels)
 
-        # Get detector type of the specified channels
+        # Get resolution of the specified channels
         if hasattr(channels, '__iter__'):
-            return [self._hist_bin_edges[ch] for ch in channels]
+            return [self._resolution[ch] for ch in channels]
         else:
-            return self._hist_bin_edges[channels]
+            return self._resolution[channels]
+
+    def hist_bins(self, channels=None, nbins=None, log=False):
+        """
+        Get a set of histogram bins the specified channel(s).
+
+        These cover the range specified in ``FCSData.range(channels)`` with
+        a number of bins `nbins`, either linearly or logarithmically
+        spaced.
+
+        Parameters
+        ----------
+        channels : int, str, list of int, list of str
+            Channel(s) for which to get the histogram bins. If None, return
+            a list with bins for all channels, in the order of the
+            `channels` attribute.
```

Comment:
We could do "in the order of `FCSData.channels`". I'm not completely convinced this would make sense for someone reading the documentation, but it is better than what we have now, I think.